start-up/setting-up-reaction-roles.md

```diff
@@ -13,43 +13,37 @@ Before beginning, make sure of the following things:
 
 ## Creating a group
 
-A group is a collection of emoji to role pairs and settings. Groups are independent for each other. After setting the appropriate pairs, you can apply the group to messages so users can start interacting with the bot and gain roles.
+A group is a collection of emoji to role pairs and settings. Groups are independent for each other. After setting the appropriate pairs, you can apply the group to messages so members can use the reactions to manage their roles.
```

Comment:
"_for_ each other" is unsound. Probably my fault when I went over this section originally.

Reply:
> Each group operates independently.